### DISCUSSION TITLE
loadMembers().get() not working for me :(

### DISCUSSION THREAD

**Discussion by Zeloslaw** [discord-jda/JDA#2246] [Questions and Help]
Hello,
I want to receive information about the number of all members of my discord. For this, I wrote the command:
```
Guild guild= jda.getGuildById(GUILD_ID);
List<Member> members = guild.loadMembers().get();
```
I turned on GatewayIntent.GUILD_MEMBERS:
```
jda = JDABuilder.createDefault(botToken)
                    .enableIntents(GatewayIntent.GUILD_MEMBERS)
                    .setMemberCachePolicy(MemberCachePolicy.ALL)
                    .build();
```
In application dashboard under the Privileged Gateway Intents section, i set enable SERVER MEMBERS INTENT and MESSAGE CONTENT INTENT.
Unfortunately it still doesn't work. Error message:
```
[main] INFO net.dv8tion.jda.api.JDA - Login Successful!
[JDA MainWS-ReadThread] INFO net.dv8tion.jda.internal.requests.WebSocketClient - Connected to WebSocket
[JDA MainWS-ReadThread] INFO net.dv8tion.jda.api.JDA - Finished Loading!
java.lang.UnsupportedOperationException: Blocking operations are not permitted on the gateway thread
	at net.dv8tion.jda.internal.utils.concurrent.task.GatewayTask.get(GatewayTask.java:96)
	at Main.onMessageReceived(Main.java:277)
```
I found in the documentation:
`You MUST NOT use blocking operations such as Task.get()! The response handling happens on the event thread by default.`
But how do I check it and apply it?

What am I doing wrong?

**Comment by HardstylesDev** [comment]
You'd use the onSuccess consumer, it'd look something like this
```
guild.loadMembers().onSuccess(members -> {
    // members is a List<Member> containing all members in the guild
});
```
